code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -926,7 +929,7 @@ Pass a positive integer as an argument to override a bot's default speed.
 			if(paicard)
 				to_chat(usr, span_notice("You eject [paicard] from [initial(src.name)]."))
 				ejectpai(usr)
-	return
+	return FALSE
 
 /mob/living/simple_animal/bot/update_icon_state()
 	icon_state = "[initial(icon_state)][get_bot_flag(BOT_MODE_ON)]"
```

Comment:
I understand it, but `get_bot_flag` depresses me

Reply:
It was just so I wouldn't have to do `` ? "on" : "off"``, I didn't know of any better suggestions